Library/Formula/mighttpd2.rb

```diff
@@ -16,14 +15,9 @@ class Mighttpd2 < Formula
 
   depends_on "ghc" => :build
   depends_on "cabal-install" => :build
-  depends_on "gmp"
```

Comment:
Oh!  I thought you had already merged that issue for some reason.  😕  I'll put the dependency back.